taskw/fields/depends.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import
+
+from distutils.version import LooseVersion
+
+import re
+import uuid
+
+from .base import DirtyableList, Field
+
+
+class DependsField(Field):
+    def deserialize(self, value):
+        if not value:
+            return DirtyableList([])
+
+        try:
+            # In task-2.5, this moved from a comma-separated string to a real list.
```

Comment:
I can't find any evidence of this and I wonder if this is actually referencing the on-disk format, possibly?

Looking at the docs, the value of this field is indeed a string: https://taskwarrior.org/docs/design/task.html#attr_depends

That all being said -- the weirdness I see on Inthe.AM is that there's a JSON-serialized list stored in this text field :shrug:.